mpy-cross/mpconfigport.h

```diff
@@ -30,6 +30,12 @@
 #define MICROPY_PERSISTENT_CODE_LOAD (0)
 #define MICROPY_PERSISTENT_CODE_SAVE (1)
 
+#if defined(__i386__) || defined(__x86_64__) || defined(_WIN32) || defined(__unix__) || defined(__APPLE__)
```

Comment:
The whole thing should be guarded by `#ifndef MICROPY_PERSISTENT_CODE_SAVE_FILE ` otherwise it cannot be overridden as it would lead to PP redefinition.